content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -143,214 +90,82 @@ Once you have a Linux-based Kubernetes control-plane ("Master") node you are rea
-            "Url" : "https://dl.k8s.io/v1.15.0/kubernetes-node-windows-amd64.tar.gz"  // Direct URL pointing to Kubernetes node binaries tarball
-        },
-        "ControlPlane" : {  // Contains values associated with Kubernetes control-plane ("Master") node
-            "IpAddress" : "kubemasterIP",  // IP address of control-plane ("Master") node
-            "Username" : "localadmin",  // Username on control-plane ("Master") node with remote SSH access  
-            "KubeadmToken" : "token",  // Kubeadm bootstrap token
-            "KubeadmCAHash" : "discovery-token-ca-cert-hash"  // Kubeadm CA key hash
-        },
-        "KubeProxy" : {  // Contains values for Kubernetes network proxy configuration
-            "Gates" : "WinOverlay=true"  // Comma-separated key-value pairs passed to kube-proxy feature gate flag
-        },
-        "Network" : {  // Contains values for IP ranges in CIDR notation for Kubernetes networking
-            "ServiceCidr" : "10.96.0.0/12",  // Service IP subnet used by Services in CIDR notation
-            "ClusterCidr" : "10.244.0.0/16"  // Cluster IP subnet used by Pods in CIDR notation 
-        }
-    },
-    "Install" : {  // Contains values and configurations for Windows node installation
-        "Destination" : "C:\\ProgramData\\Kubernetes"  // Absolute DOS path where Kubernetes will be installed on the Windows node
-    }
-}
-  ```
-
 {{< note >}}
-Users can generate values for the `ControlPlane.KubeadmToken` and `ControlPlane.KubeadmCAHash` fields by running  `kubeadm token create --print-join-command` on the Kubernetes control-plane ("Master") node.
+All code snippets in Windows sections are to be run in a PowerShell environment
+with elevated permissions (Administrator) on the Windows worker node.
 {{< /note >}}
 
-1. Install containers and Kubernetes (requires a system reboot)
-
-Use the previously downloaded [KubeCluster.ps1](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/KubeCluster.ps1) script to install Kubernetes on the Windows Server container host:
-
-  ```PowerShell
-    .\KubeCluster.ps1 -ConfigFile .\Kubeclustervxlan.json -install 
-  ```
-  where `-ConfigFile` points to the path of the Kubernetes configuration file.
+1. Install wins, kubelet, and kubeadm.
 
+   ```PowerShell
+   curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/PrepareNode.ps1
+   .\PrepareNode.ps1 -KubernetesVersion v1.17.0
+   ```
 {{< note >}}
-In the example below, we are using overlay networking mode. This requires Windows Server version 2019 with [KB4489899](https://support.microsoft.com/help/4489899) and at least Kubernetes v1.14 or above. Users that cannot meet this requirement must use  `L2bridge` networking instead by selecting `bridge` as the [plugin](https://github.com/kubernetes-sigs/sig-windows-tools/blob/master/kubeadm/v1.15.0/Kubeclusterbridge.json#L18) in the configuration file.  
+Kubernetes version must be >= 1.17.0
```

Comment:
Adding a `version-check` shortcode earlier should mean you don't need to repeat this.